physical_property/logP/example_submission_file/logP-DanielleBergazinExampleFile-1.csv

```diff
@@ -9,40 +9,66 @@
 #
 # PREDICTION SECTION
 #
-# It is mandatory to submit  water to octanol (ΔG_octanol - ΔG_water) transfer free energy (TFE) predictions for all 22 molecules. Incomplete submissions will not be accepted.
+# It is mandatory to submit  water to octanol (ΔG_octanol - ΔG_water) transfer free energy (TFE) predictions for all 22 molecules.
+# Incomplete submissions will not be accepted.
 # The energy units must be in kcal/mol.
 # Please report TFE standard error of the mean (SEM) and TFE model uncertainty.
 #
 # The data in each prediction line should be structured as follows:
-# Molecule ID, TFE, TFE SEM, TFE model uncertainty
+# ID tag, Molecule ID, TFE, TFE SEM, TFE model uncertainty
+#
+# Your transfer free energy prediction for the neutral form does NOT have to be `microstate000` (the challenge provided microstate).
+# If you use a microstate other than the challenge provided microstate, please use a different `Molecule ID` in the form of `SMXX_extra001` (number can vary) and include the smiles string in your methods description.
```

Comment:
I think you have to clarify "ID tag" and "molecule ID" here. Not defined.

Reply:
Updated